notebooks/person-counting-webcam/README.md

```diff
@@ -0,0 +1,27 @@
+# Person Counting System using YOLOv8 and OpenVINO
+
+[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/person-counting-webcam/person-counting.ipynb)
+
+In this project, we utilized the YOLOv8 Object Counting class to develop a real-time person counting system using the YOLOv8 object detection model and tracking, optimized for Intel's OpenVINO toolkit to enhance inferencing speed. This system effectively monitors the number of individuals entering and exiting a room, leveraging the optimized YOLOv8 model for accurate person detection under varied conditions.
+
+By utilizing the OpenVINO runtime on Intel hardware, the system achieves significant improvements in processing speed, making it ideal for applications requiring real-time data, such as occupancy management and traffic flow control in public spaces and commercial settings.
+
+References:
+
+- YOLOv8 Object counting documentation: <a href="https://docs.ultralytics.com/guides/object-counting/" target="_blank">https://docs.ultralytics.com/guides/object-counting/</a>
+- OpenVINO Jupyter Notebooks: <a href="https://github.com/openvinotoolkit/openvino_notebooks/" target="_blank">https://github.com/openvinotoolkit/openvino_notebooks/</a>
+
+<div align="center"><img src="person-count.gif" width=900/></div>
+
+
+## Performance
+
+In this clip, you can see the difference (Inference time and FPS) between running YOLOv8 natively with PyTorch vs optimized with OpenVINO.
+
+<div align="center"><img src="https://github.com/antoniomtz/people-counting-yolov8-openvino/raw/main/optimized.gif" width=900/></div>
```

Comment:
is it expected that this link lead to your repository?

Reply:
The file is large and didn't want to include it in the repo. We can keep it like this or where should I upload assets?

Reply:
no, staying as is ok, I just asking in case if it is typo or you forgot to update it